src/ts/helpers/polyfills.ts

```diff
@@ -0,0 +1,52 @@
+// Promise polyfill
+import Promise from 'promise-polyfill';
+if (!window['Promise']) window['Promise'] = Promise;
```

Comment:
I'm using polyfills only in `app.ts` (frontend) and it's not used on server side, but I added additional check anyway.